protocols/luci-proto-ipv6/htdocs/luci-static/resources/protocol/dhcpv6.js

```diff
@@ -30,6 +30,8 @@ return network.registerProtocol('dhcpv6', {
 		o.value('60');
 		o.value('64');
 		o.default = 'auto';
+		o = s.taboption('general', form.Flag, 'extendprefix', _('Extend prefix'), _('Accept a /64 prefix via SLAAC and extend it as prefix delegation on downstream interface (RFC 7278)'));
```

Comment:
Add a line above the option, so it's not together with `		o.default = 'auto';`

```js
		o = s.taboption('general', form.Flag, 'extendprefix', _('Extend prefix'), _('Extend 3GPP WAN interface /64 prefix via PD to LAN (RFC 7278)'));
```

Update also the i18n strings.

Reply:
it's not DHCPv6 Prefix Delegation. This option works by obtaining the prefix from SLAAC and using it as IPv6 PD. So saying "via PD" isn't correct.

Reply:
I didn't imply it was. Prefixes are not obtained via SLAAC. But a single /128 is. 

How is the prefix delegated on the lan side?

Reply:
> I didn't imply it was. Prefixes are not obtained via SLAAC. But a single /128 is.
> 
> How is the prefix delegated on the lan side?

should "3GPP" be replaced with something more general? In my case it's not a 3GPP network but a wired broadband connection.